internal/healthcheck/healthcheck.go

```diff
@@ -88,8 +89,24 @@ func (c *Check) HandleReadiness(w http.ResponseWriter, _ *http.Request) {
 		return
 	}
 
-	err := c.proxy.CheckConnections(ctx)
-	if err != nil {
+	var minReady int
```

Comment:
It is confusing that when the `min-ready` request url param is not set or has an error, then minReady = 0. If minReady = 0. It would be clearer if minReady = n by default.

Reply:
I've updated this slightly to be more readable.